drivers/video/video_mcux_smartdma.c

```diff
@@ -271,6 +272,7 @@ static int nxp_video_sdma_get_format(const struct device *dev, struct video_form
 	}
 
 	fmt->pitch = fmt->width * video_bits_per_pixel(fmt->pixelformat) / BITS_PER_BYTE;
+	fmt->size = fmt->pitch * SDMA_LINE_COUNT;
```

Comment:
This is the minimum size that the driver author can allocate, but has the merit of allowing `fmt->size` to be accurate, and not require any wrapper in `video_set_format()`. Good idea!